templates/products.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```

Comment:
I don't know if this file will give you exactly what you want, but its probably a good place to start.